.github/actions/setup-sql-server/Initialize-SqlServer.ps1

```diff
@@ -5,14 +5,14 @@ if ($Env:RUNNER_OS -eq 'Windows')
 else
 {
     $dockerRunSwitches = @(
-        '--name', 'sql2019'
+        '--name', 'sql2022'
         '--env', 'ACCEPT_EULA=Y'
         '--env', 'SA_PASSWORD=Password1!'
         '--publish', '1433:1433'
-        '--detach', 'mcr.microsoft.com/mssql/server:2019-latest'
+        '--detach', 'mcr.microsoft.com/mssql/server:2022-latest'
     )
 
-    docker pull mcr.microsoft.com/mssql/server &&
+    docker pull mcr.microsoft.com/mssql/server:2022-latest &&
     docker run @dockerRunSwitches &&
-    docker exec --user 0 sql2019 bash -c 'mkdir /data; chmod 777 /data --recursive; chown mssql:root /data'
+    docker exec --user 0 sql2022 bash -c 'mkdir /data; chmod 777 /data --recursive; chown mssql:root /data'
```

Comment:
@Piedone do we need to use the SQLServer version as the container name? Can we use for example `uitt-sqlserver` here and `Invoke-SolutionTests.ps1`:L25. I think not using version makes everything a bit better.

Reply:
The container name is used as an ENV variable that `SqlServerManager` will use so if we change that here we need to change the ENV var aswell. So do we want to change it or not?